src/components/Registration/StartDate.vue

```diff
@@ -80,33 +79,46 @@ export default class StartDate extends Mixins(DateMixin) {
 
   /** Validations rules for start date field. */
   get startDateRules (): Array<Function> {
-    // apply rules when app validations are triggered
-    if (this.getShowErrors) {
-      return [
-        (v: string) => !!v || 'Business start date is required',
-        (v: string) =>
-          RuleHelpers.DateRuleHelpers
-            .isBetweenDates(this.startDateMin,
-              this.startDateMax,
-              v) ||
-          `Date should be between ${this.dateToPacificDate(this.startDateMin, true)} and
-         ${this.dateToPacificDate(this.startDateMax, true)}`
-      ]
-    }
+    return [
+      (v: string) => !!v || 'Business start date is required',
+      (v: string) =>
+        RuleHelpers.DateRuleHelpers
+          .isBetweenDates(this.startDateMin,
+            this.startDateMax,
+            v) ||
+        `Date should be between ${this.dateToPacificDate(this.startDateMin, true)} and
+        ${this.dateToPacificDate(this.startDateMax, true)}`
+    ]
   }
 
   private startDateHandler (dateString: string): void {
     this.dateText = dateString
     this.setRegistrationStartDate(dateString)
+    this.emitValid()
   }
 
   @Watch('getShowErrors')
   validateForm (): void {
     (this.$refs.startDateRef as any).validateForm()
   }
+
+  @Emit('valid')
+  private emitValid (): boolean {
+    return !!this.dateText
+  }
 }
 </script>
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+
+.start-date-title {
+  font-weight: bold;
+  color: $gray9;
+}
+
+// remove extra space taken by error message
+::v-deep .v-text-field__details {
+  margin-bottom: -8px !important;
+}
```

Comment:
To prevent some components from having tons of whitespace below them, sometimes you can use "hide-details", and sometimes you have to do this. The issue with hide-details is it doesn't show error messages, while this trick does (but it does run into the whitespace to the next element, which is usually OK).